projects/bulma-app/src/assets/docs/input/input-text.md

```diff
@@ -2,19 +2,36 @@
 
 #### The `<input>` tag specifies an input field where the user can enter data.
 
-##### The following type attributes are supported:
+#### Following are the different type of attributes:
 
 `type="text"`
 `type="password"`
 `type="email"`
 `type="tel"`
 
 To display a Input component in your app and verify that everything works.
-You need to import the BulmaInputModule that you want to display by adding the following lines to your app.module.ts file.
+You need to import the `BulmaInputModule` that you want to display by adding the following lines to your `app.module.ts` file.
 
 ```javascript
 import { BulmaInputModule} from 'ngx-bulma'
  @NgModule({
    imports: [BulmaInputModule]
  });
 ```
+
+### Basic
+
+To use the Input component in your Angular application.
+Add `buInput` in Input tags.
+
+| Directive | Description                           |     |     |     |
+| --------- | ------------------------------------- | --- | --- | --- |
+| `buInput` | For initializing buInput in your HTML |     |     |     |
```

Comment:
I am not sure why we have 3 extra columns in this table. I have seen extra columns in all the tables in this PR. @santoshyadav198613 - Do we need to have extra columns?